core/services/ardupilot_manager/mavlink_proxy/test_all.py

```diff
@@ -63,17 +63,39 @@ def test_mavproxy() -> None:
     assert mavproxy.set_logdir(pathlib.Path(".")), "Local path as MAVProxy log directory failed."
     assert re.search(r"\d+.\d+.\d+", str(mavproxy.version())) is not None, "Version does not follow pattern."
 
-    endpoint_1 = Endpoint("udpout", "0.0.0.0", 14551)
+    endpoint_1 = Endpoint("udpin", "0.0.0.0", 14551)
     endpoint_2 = Endpoint("udpout", "0.0.0.0", 14552)
+    endpoint_3 = Endpoint("tcpin", "0.0.0.0", 14553)
+    endpoint_4 = Endpoint("tcpout", "0.0.0.0", 14554)
+    endpoint_5 = Endpoint("serial", "/dev/radiolink", 57600)
     assert mavproxy.add_endpoint(endpoint_1), "Failed to add first endpoint"
     assert mavproxy.add_endpoint(endpoint_2), "Failed to add second endpoint"
+    assert mavproxy.add_endpoint(endpoint_3), "Failed to add third endpoint"
+    assert mavproxy.add_endpoint(endpoint_4), "Failed to add fourth endpoint"
+    assert mavproxy.add_endpoint(endpoint_5), "Failed to add fifth endpoint"
```

Comment:
Well, it's time to use for loops here and avoid all duplicated code.

Reply:
This is one of those funny moments when I look at it and think "why the hell is that like this?"